geomesa-cassandra/geomesa-cassandra-tools/conf/application.conf

```diff
@@ -0,0 +1,31 @@
+geomesa {
```

Comment:
is this any different than the common examples? it would be nice if we could keep them all the same. We could (should) move ingest_csv.sh into each tools project.